e2etest/bgp_container.go

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+Copyright 2021 The MetalLB Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+// https://github.com/kubernetes/kubernetes/blob/92aff21558831b829fbc8cbca3d52edc80c01aa3/test/e2e/network/loadbalancer.go#L878
+
+package e2e
```

Comment:
I think this deserves to be in its own package.

I'd choose `frr` for the container operations, `frrconfig` or just `config` under frr for the config operations